src/components/AddAsset.tsx

```diff
@@ -56,8 +56,8 @@ export const AddAsset = ({ onClose }: { onClose: () => void }) => {
       );
       setLocalStatus(ActionStatus.SUCCESS);
     } catch (e) {
-      log.error({ title: e.toString() });
-      setErrorMessage(e.toString());
+      log.error({ title: e.message });
```

Comment:
@piyalbasu Do you think we should make a helper function to display `error.message` or `error.toString()` if `message` is not available. Errors that we are throwing should have a `message` prop, but there might be cases where it comes from somewhere else and doesn't have it? 🤔

Reply:
I think that's a good idea! I'm running into a similar issue in Freighter where `e` doesn't always have a `message` attached to it